# Author: Ehsan Moravveji (VSC - KU Leuven)
# Purpose: the 'license' filed in the pyproject.toml file did not
#          comply with the expected format, which is fixed here.

diff -ruN dulwich-0.24.5-orig/pyproject.toml dulwich-0.24.5/pyproject.toml
--- dulwich-0.24.5-orig/pyproject.toml	2025-10-16 11:24:53.000000000 +0200
+++ dulwich-0.24.5/pyproject.toml	2025-10-21 10:37:28.078750000 +0200
@@ -7,7 +7,7 @@
 description = "Python Git Library"
 readme = "README.rst"
 authors = [{name = "Jelmer Vernooij", email = "jelmer@jelmer.uk"}]
-license = "Apache-2.0 OR GPL-2.0-or-later"
+license = { text = "Apache-2.0 OR GPL-2.0-or-later" }
 keywords = ["vcs", "git"]
 classifiers = [
     "Development Status :: 4 - Beta",
@@ -29,7 +29,7 @@
     'typing_extensions >=4.0 ; python_version < "3.12"',
 ]
 dynamic = ["version"]
-license-files = ["COPYING"]
+#license-files = ["COPYING"]
 
 [project.urls]
 Homepage = "https://www.dulwich.io/"
